test/graphics/graphics-device.test.mjs

```diff
@@ -1,4 +1,4 @@
-import { GraphicsDevice } from '../../src/graphics/webgl/graphics-device.js';
```

Comment:
The correct thing to do here is `git mv` this test into `test/graphics/webgl/webgl-graphic-device.mjs`.

Reply:
nice one, done, thanks